### PR TITLE
[FAN-170] Unified Comicbooks

### PR DESCRIPTION
**Pull request by Denise54321**

### CODE REVIEW

File: src/modules/overview/actions/fetchItemsByFilter.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export enum ItemsChannel {
   all = 'All',
   fineArts = 'FineArts',
   sports = 'Sports',
-  comics = 'Conicbooks',
```
Comment (Antroll) [inline src/modules/overview/actions/fetchItemsByFilter.ts:8]: Are you sure? Was that typo error fixed on backend?
This is the request parameter for API.

Comment (BigBadAlien) [reply src/modules/overview/actions/fetchItemsByFilter.ts:8]: Is it typo on our side or on the backend?

File: src/modules/overview/actions/fetchItemsByFilter.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export enum ItemsChannel {
...
+  Comicbooks = 'Comicbooks',
```
Comment (BigBadAlien) [inline src/modules/overview/actions/fetchItemsByFilter.ts]: camelCase

Comicbooks -> comicBooks

at least on our side

**Review by BigBadAlien — APPROVED** [review]

File: src/modules/overview/actions/fetchItemsByFilter.ts (hunk 1 above)
Comment (Denise54321) [reply src/modules/overview/actions/fetchItemsByFilter.ts:8]: In fact we need keep the parameters the same in both creat method and query method. But document isn't the sam. Confirmed with the guy to use the Comicbooks

**Pull request merged**